client/game/SocialExposure.jsx

```diff
@@ -32,9 +32,11 @@ export default class SocialExposure extends React.Component {
     return (
       <>
         <SlidersPrediction disabled={true} {...this.props} />
```

Comment:
As per @npaton commented earlier, i think we should not show the slider as well, right?